go-controller/pkg/ovn/secondary_layer2_network_controller_test.go

```diff
@@ -575,3 +503,117 @@ func nodeCIDR() *net.IPNet {
 		Mask: net.CIDRMask(24, 32),
 	}
 }
+
+func setupFakeOvnForLayer2Topology(fakeOvn *FakeOVN, initialDB libovsdbtest.TestSetup, netInfo secondaryNetInfo, testNode *v1.Node, podInfo testPod, pod *v1.Pod) error {
+	var err error
```

Comment:
nit: no need to, you're initializing this below, when you create the NAD.

Reply:
right. DONE